Add support for volume level

DestruktorGitHub opened the pull request:
I also tried to implement the ability to set the volume level, but it turned out that this is no longer supported in newer versions of iOS, so I left only the functions responsible for displaying the current Airplay volume, which may make the code look a little messy.

DestruktorGitHub committed:
Add Volume Level

DestruktorGitHub committed:
Prevent error on slider interaction

A minor fix that prevents an error from being displayed when the user tries to use the slider.

charlesomer commented:
This looks like it would close: #27 too. Just tested it on my setup and at first glance it appears to work. Nice! Although I imagine it's going to confuse people when setting the volume via HA doesn't work so some documentation might be needed 😄

parautenbach commented:
Thank you for the PR! 

> Although I imagine it's going to confuse people when setting the volume via HA doesn't work so some documentation might be needed

Agreed.

Please also bump the version.

DestruktorGitHub committed:
Add Polish translation for config flow

DestruktorGitHub committed:
Update README.md & CHANGELOG.md

DestruktorGitHub commented:
Okay, I added a Polish translation for the config flow, changed the README file to explain the current situation (why remote control from HA is currently not working), and listed the new features in the CHANGELOG file.
